src/app/app.component.html

```diff
@@ -1,2 +1,8 @@
-
 <router-outlet></router-outlet>
+<h1  class="hideMeText" style="text-align: center; padding: 3%">Welcome to Earth BnB Page</h1>
```

Comment:
This might be subject to change in future updates, but I think you're trying to give us a "main" page. I'll write a route for main